lib/verdict/experiment.rb

```diff
@@ -162,6 +162,13 @@ def store_assignment(assignment)
     assignment
   end
 
+  def cleanup
+    case @storage
+    when Verdict::Storage::RedisStorage
```

Comment:
Why did we choose to switch on the type here instead of letting polymorphism do its thing?

Reply:
I would be inclined to stub a method in base to raise `NotImplementedError`, but maybe that's actually what we want to do instead of doing nothing?

Reply:
I definitely prefer that over the switch here 👍